docs/configuration/configuration.md

```diff
@@ -2283,12 +2287,29 @@ The following meta labels are available on targets during [relabeling](#relabel_
 * `__meta_stackit_status`: The current status of the server.
 * `__meta_stackit_power_status`: The power status of the server.
 
+#### `postgres_flex`
+
+The postgres_flex role dynamically discovers scrape targets using the [PostgresFlex API](https://docs.api.eu01.stackit.cloud/documentation/postgres-flex-service/version/v2#tag/instance).
+
+To enable this scrape configuration, a valid Authorization Bearer Token must be provided. Refer to [this example configuration](../../documentation/examples/prometheus-stackit.yml) for guidance.
```

Comment:
A good hint would be useful here that the recommended STACKIT Service Account Key is not supported here.